lib/check_request.go

```diff
@@ -0,0 +1,32 @@
+// Copyright 2020 Security Scorecard Authors
+//
```

Comment:
file names with `_` are primarily only for `test` files and for assign platform/arch-only code,
```
➜ cd $GOROOT/src/pkg/math/
➜ ls sqrt*s
sqrt_386.s  sqrt_amd64p32.s  sqrt_amd64.s  sqrt_arm.s
```

Reply:
i dont think that is correct, see e.g. 
https://github.com/kubernetes/kubernetes/tree/master/pkg/kubelet/dockershim
docker_logs.go
docker_sandbox.go
docker_sandbox_test.go

_test.go is a suffix for tests, but there is no rule against using _ for file names.

Reply:
Ok. But I am basing my comments based standard library. 

My recommendation was to avoid that as much as possible.

Reply:
yes i dont disagree, avoid when possible, but fine to use it otherwise.

Reply:
Will continue to keep it as _ filenames for now, since they help with readability. Can change it in future if needed.